src/ert/storage/local_ensemble.py

```diff
@@ -488,7 +419,12 @@ def get_failure(self, realization: int) -> Optional[_Failure]:
             )
         return None
 
-    def get_ensemble_state(self) -> List[RealizationStorageState]:
+    def refresh_ensemble_state(self) -> None:
```

Comment:
Maybe this should be a more generic: `refresh`, and should probably be part of the `LocalStorage` `refresh`, or that might be implicit?

Reply:
I think having it be a part of refresh in `LocalStorage` makes sense, renaming to `refresh` makes sense if we add some more to it, was thinking to add it when combining datasets.